Spigot-Server-Patches/0551-Add-a-way-to-get-translation-keys-for-blocks-entitie.patch

```diff
@@ -0,0 +1,110 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: MeFisto94 <MeFisto94@users.noreply.github.com>
+Date: Tue, 11 Aug 2020 19:16:09 +0200
+Subject: [PATCH] Add a way to get translation keys for blocks, entities and
+ materials
+
+
+diff --git a/src/main/java/net/minecraft/server/Block.java b/src/main/java/net/minecraft/server/Block.java
+index 81725611d5b8c10c05d561984f92dc67349900ec..a5c4bb06dc2a27121e638c68cdf05c948c5334d4 100644
+--- a/src/main/java/net/minecraft/server/Block.java
++++ b/src/main/java/net/minecraft/server/Block.java
+@@ -282,6 +282,7 @@ public class Block extends BlockBase implements IMaterial {
+         return !this.material.isBuildable() && !this.material.isLiquid();
+     }
+ 
++    public String getDescriptionId() { return i(); } // Paper - OBFHELPER
+     public String i() {
+         if (this.name == null) {
+             this.name = SystemUtils.a("block", IRegistry.BLOCK.getKey(this));
+diff --git a/src/main/java/net/minecraft/server/EntityTypes.java b/src/main/java/net/minecraft/server/EntityTypes.java
+index 18a806ebbf092b904983691529ce5edf2da4e6db..c252ffe6044ad3266f74c648911709723adc490c 100644
+--- a/src/main/java/net/minecraft/server/EntityTypes.java
++++ b/src/main/java/net/minecraft/server/EntityTypes.java
+@@ -149,6 +149,7 @@ public class EntityTypes<T extends Entity> {
+     public static Optional<EntityTypes<?>> a(String s) {
+         return IRegistry.ENTITY_TYPE.getOptional(MinecraftKey.a(s));
+     }
++    public static Optional<EntityTypes<?>> getByName(String name) { return a(name); } // Paper - OBFHELPER
```

Comment:
Needs to be brought down a line, whitespace can matter in creating conflicts.

Reply:
this is an obf helper for the method above, I moved it above the method, like usual